src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java

```diff
@@ -0,0 +1,39 @@
+package harshadBJun20.assignment34;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/*Assignment - 34 :  25th July 2020
+ * Program 2 :
+Display on which index Techno is available, Replace all Techno with Credits from given ArrayList, and count total number of Credits.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4 */
+
+public class FindAndReplaceNameFromArrayList {
+	
+	void findAndReplaceName(ArrayList<String> namesList, String orgName, String replaceName) {
+		ArrayList<Integer> indexOfWord= new ArrayList<Integer>();
```

Comment:
Declaring just a counter variable, would also serve the purpose. But this is also fine.

Reply:
ok Aastha Thanks. I will do necessary changes and update the PR.

Reply:
Changes Done and updated PR as per review comment. Please review. Thanks